intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -209,6 +209,32 @@ \section{Defining and submitting your job}
 problem that prevented a successful run. The standard output file will contain
 the results of your calculation (here, the output of the perl script)
 
+\ifgent
+\subsection{When will my job start?}
+\label{subsec:priority}
+In practice it's impossible to predict when your job(s) will start,
+since most currently running jobs will finish before their requested walltime
+expires, new jobs by may be submitted by other users that are assigned a higher
+priority than your job(s).
+
+The clusters use a fair-share sheduling policy. There is no guarantee on when a
+job will start, since it depends on a number of factors. One of these factors is
```

Comment:
If you say `One of these factors`, you'll need to explain the others too (I guess those are: how busy the cluster is, how many workernodes are active, the resources (e.g. number of cores, memory) provided by each workernode, etc.).